packages/common/src/types/schema.ts

```diff
@@ -1469,9 +1501,10 @@ export type InvoiceLineNode = {
   note?: Maybe<Scalars['String']>;
   numberOfPacks: Scalars['Int'];
   packSize: Scalars['Int'];
+  pricing: PricingNode;
```

Comment:
this is a bit weird but might be my fault. I suggested that the pricing structure for entities be consistent. Trying to say that `InvoiceNode` had an `InvoicePricingNode` which had ` {total, subtotal, stockTotal, serviceTotal, tax}` but on `InvoiceLineNode` it was all flattened.

So in addition to the flattened fields there has been the same object added on an invoice line (where having serviceTotal, stockTotal and totalTotal fields is not helpful)

This required adding the pricing node where `InvoiceLineNode` was being used (`useNextItem.test`) - though that test needs tidying in general